2wr-app/src/router/index.js

```diff
@@ -32,6 +34,17 @@ const routes = [
       component: EmergencyKitCreatePage,
       meta: { requiresAuth: true }
     },
+    {
+      path: '/prepare/hazardhunt',
+      name: 'hazardhunt',
+      component: HazardHuntListing,
+      meta: { requiresAuth: false }
+    },    {
+    path: '/prepare/hazardinfo',
+      name: 'hazardinfo',
+      component: HazardInfoListing,
+      meta: { requiresAuth: false }
```

Comment:
```suggestion
      meta: { requiresAuth: true }
```

Same as above

Reply:
Thanks for the review. I can make that change. I had asked about authentication for these pages in the discussion of issue #30. It seemed like this was not sensitive or personalized information so that made sense to me. Another option to remove the authentication from the API implementation for the hazard data. Thoughts?